### PR TITLE
ATS-380 : Update AI amp version to 1.0.0

### PR DESCRIPTION
**Pull request by CezarLeahu**
Once merged, this change will be cherry-picked into:
 * **support/SP/6.1.N**
 * **master**

### CODE REVIEW

File: distribution/src/main/resources/VERSIONS.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ This file lists the recommended components for this Service Pack. Use it along w
 | Alfresco Google Docs Integration | @alfresco.googledocs.version@ |
 | Alfresco Records Management | @alfresco.records-management.version@ |
 | Alfresco Media Management | @alfresco.mm.version@ |
-| Alfresco Intelligence Services | 0.4.0 |
+| Alfresco Intelligence Services | 1.0.0 |
```
Comment (montgolfiere) [inline distribution/src/main/resources/VERSIONS.md]: @alfresco.ai-repo.version@ instead ?

Comment (CezarLeahu) [reply distribution/src/main/resources/VERSIONS.md]: Yes, done.

**Comment by montgolfiere** [comment]
Apparently one test failing https://bamboo.alfresco.com/bamboo/browse/PLAT-ACS308 ?

**Comment by CezarLeahu** [comment]
We've updated the build plan variables on this branch, so that it matches the variable on **support/HF/6.1.0**. It did fix the tests, but the build is still failing with other errors:
> No failed tests found, a possible compilation error occurred.

The **support/SP/6.1.N** branch build is failing in a similar manner, and I suspect the **support/HF/6.1.0** branch build would also fail, though it's currently disabled.

**Pull request merged**